tests/Unit/Mail/FeedbackRequestTest.php

```diff
@@ -29,6 +29,7 @@ public function email()
 
             return $mail->feedback->id === $feedback->id &&
                 $mail->hasFrom($feedback->location->contact_email, $feedback->location->title) &&
+                //Todo: require Bookings
```

Comment:
https://github.com/tipoff/bookings/pull/47 is blocking this since `$feedback->participant->email` depends on the Participant model email field.